### PR TITLE
2297 add otelcol exporter jaeger component

### LINKED ISSUE #2297: Flow: otelcol.exporter.jaeger component
Expose the OpenTelemtry Collector `jaeger` exporter as a Flow component called `otelcol.exporter.jaeger`. This component is from the `otelcol-contrib` distribution.

### PR DESCRIPTION
**Pull request by erikbaranowski**
#### PR Description

Introduce an otelcol.exporter.jarger component which wraps around the upstream OpenTelemetry Jaeger exporter.

#### Which issue(s) this PR fixes

Closes #2297 

#### PR Checklist

- [x] CHANGELOG updated
- [x] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: docs/sources/flow/reference/components/otelcol.exporter.jaeger.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+otelcol.exporter.jaeger "jaeger" {
+    client {
+        endpoint = "jaeger:14250"
+        tls {
+            insecure             = true
+            insecure_skip_verify = true
+        }
+    }
+}
+```
```
Comment (rfratto) [inline docs/sources/flow/reference/components/otelcol.exporter.jaeger.md:174]: Can you change this to show a pipeline like `otelcol.receiver.otlp` -> `otelcol.processor.batch` -> `otelcol.exporter.jaeger`?

**Review by rfratto — APPROVED** [review]

File: docs/sources/flow/reference/components/otelcol.exporter.jaeger.md (hunk 1 above)
Comment (erikbaranowski) [reply docs/sources/flow/reference/components/otelcol.exporter.jaeger.md]: added

File: CHANGELOG.md [hunk 2]
```diff
@@ -43,6 +43,8 @@ Main (unreleased)
   - `loki.source.syslog` listens for Syslog messages over TCP and UDP
     connections and forwards them to other `loki` components. (@tpaschalis)
   - `loki.source.windowsevent` reads logs from Windows Event Log. (@mattdurham)
+  - `otelcol.exporter.jaeger` forwards OTLP-formatted data to compatible `jaeger`
+    receivers. (@erikbaranowski)
```
Comment (rfratto) [inline CHANGELOG.md:47]:
```suggestion
  - `otelcol.exporter.jaeger` forwards OpenTelemetry data to a Jaeger server.
    (@erikbaranowski)
```

File: docs/sources/flow/reference/components/otelcol.exporter.jaeger.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
+---
+aliases:
+- /docs/agent/latest/flow/reference/components/otelcol.exporter.jaeger
+title: otelcol.exporter.jaeger
+---
+
+# otelcol.exporter.jaeger
+
+`otelcol.exporter.jaeger` accepts telemetry data from other `otelcol` components
+and writes them over the network using the OTLP gRPC protocol.
```
Comment (rfratto) [inline docs/sources/flow/reference/components/otelcol.exporter.jaeger.md:10]:
```suggestion
`otelcol.exporter.jaeger` accepts telemetry data from other `otelcol` components
and writes them over the network using the Jaeger protocol. 
```

**Pull request merged**